tests/cocoapod/gRPCSample/Podfile

```diff
@@ -6,7 +6,6 @@ install! 'cocoapods', :deterministic_uuids => false
 GRPC_PODSPEC_ROOT = '../../../native'
 
 target 'gRPCSample' do
-  use_frameworks!
```

Comment:
This could be problematic since almost all Apple apps now include Swift and thus require `use_frameworks!`

Reply:
Swift supports static libraries, but still needs them packaged in frameworks with `use_frameworks! :linkage => :static`. 

Separate from that, and sorry I don't have the full context here, SDKs often need to support clients choosing to link dynamically or statically.